tool/mediapipe_ci_script_stable.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+DIR="${BASH_SOURCE%/*}"
+source "$DIR/ci_script_shared.sh"
+
+# flutter doctor -v
+
+declare -ar PACKAGE_NAMES=(
+    "mediapipe-core"
+)
+
+ci_package "stable" "${PACKAGE_NAMES[@]}"
+
```

Comment:
Add NL at EOF

Reply:
You'd be so proud of me. I saw those as I was opening the PR, added the newlines, forgot to stage my changes, committed nothing, and force-pushed. Yay!